packages/synapse-types/src/Table/QueryFilter.ts

```diff
@@ -44,14 +44,12 @@ export const TEXT_MATCHES_QUERY_FILTER_CONCRETE_TYPE_VALUE =
   'org.sagebionetworks.repo.model.table.TextMatchesQueryFilter'
 export type TEXT_MATCHES_QUERY_FILTER_CONCRETE_TYPE =
   typeof TEXT_MATCHES_QUERY_FILTER_CONCRETE_TYPE_VALUE
-export interface TextMatchesQueryFilter extends QueryFilter {
+export interface TextMatchesQueryFilter {
   concreteType: TEXT_MATCHES_QUERY_FILTER_CONCRETE_TYPE
   searchExpression: string
 }
 
-export interface QueryFilter {
-  concreteType:
-    | TEXT_MATCHES_QUERY_FILTER_CONCRETE_TYPE
-    | COLUMN_SINGLE_VALUE_QUERY_FILTER_CONCRETE_TYPE
-    | COLUMN_MULTI_VALUE_FUNCTION_QUERY_FILTER_CONCRETE_TYPE
-}
+export type QueryFilter =
+  | TextMatchesQueryFilter
+  | ColumnSingleValueQueryFilter
+  | ColumnMultiValueFunctionQueryFilter
```

Comment:
So much easier to work with a union of types rather than an interface that may or may not be extended